content/influxdb3/enterprise/reference/config-options.md

```diff
@@ -197,17 +195,30 @@ configuration--for example, the same bucket.
 
 ---
 
+#### cluster-id
+
+Specifies the cluster identifier that prefixes the object store path for the Enterprise Catalog. 
+This value must be different than the [`--node-id`](#node-id) value.
+
+| influxdb3 serve option | Environment variable               |
+| :--------------------- | :--------------------------------- |
+| `--cluster-id`         | `INFLUXDB3_ENTERPRISE_CLUSTER_ID`  |
+
+---
+
 #### mode
 
 Sets the mode to start the server in.
 
 This option supports the following values:
 
-- `read`
-- `read_write` _(default)_
-- `compactor`
+- `all` _(default)_
+- `ingest`
+- `query`
+- `compact`
+- `process`
```

Comment:
I remember seeing discussion that this option should accept a list, so you can pass multiple modes to a node. @peterbarnett03 can you confirm? If so, we need to know the list syntax to use for the command line option and the environment variable.

I'm wouldn't hold this PR up on this feedback, but I think we should give a description of what each mode does.